packages/reqresp/src/request/errors.ts

```diff
@@ -31,6 +31,8 @@ export enum RequestErrorCode {
   RESP_TIMEOUT = "REQUEST_ERROR_RESP_TIMEOUT",
   /** Request rate limited */
   REQUEST_RATE_LIMITED = "REQUEST_ERROR_RATE_LIMITED",
+  /** */
```

Comment:
why is this left empty, sounds useful to know later why it was introduced without looking at the PR

Reply:
yes please do a follow up PR @matthewkeil